docs/examples/django/README.rst

```diff
@@ -100,6 +99,13 @@ output similar to this one:
 The last output shows spans automatically generated by the OpenTelemetry Django
 Instrumentation package.
 
+Disabling Django Instrumentation
+--------------------------------
+
+Django's instrumentation can be disabled by setting the following environment variable.
```

Comment:
Curious as to why we would expose a way to disable instrumentation where the user can simply not instrument manually/autoinstrument? Just asking because I don't think we have this mechanism for any other instrumentation.

Reply:
This was inspired in the DataDog implementation. If you have the Xinstrumentation plugin installed in your virtual environment it will instrument the Xlibrary when you run auto instrumentation, regardless if the user wants it or not. I see that this has some value when testing stuff, since it is easier than uninstalling the Xinstrumentation plugin, but I don't find this to be a critical feature. Maybe we should follow the same pattern in the rest of the instrumentations or remove it from this one to be consistent?

Reply:
I think it is helpful when auto-instrumentation comes into play especially with the bootstrap command. The flow looks something like this:

```
pip install opentelemetry-instrumentation
opentelemetry-instrumentation-bootstrap -a=install
opentelemetry-instrument python main.py
```
Since the bootstrap command does not have any way to exclude certain instrumentations, an env var like this can help selectively disable some in case users find issues. I think instead of having this for every instrumentation, we can have a single var for the instrument command that accept a comma separated values to disable instrumentations. Something like,

```
export OTEL_PYTHON_DISABLED_INSTRUMENTATIONS=django,psycopg2
opentelemetry-instrument python main.py
```

This will allow people to temporarily disable instrumentation if they want and the feature will be consolidated in a single place (instrument command).

Reply:
Yeah consistency is what I recommend. @owais I don't have a strong opinion about this, please create an issue for the other instrumentations if you do decide to keep it.

Reply:
https://github.com/open-telemetry/opentelemetry-python/issues/1240